src/renderers/shared/ReactDebugTool.js

```diff
@@ -18,8 +19,65 @@ var ReactChildrenMutationWarningHook = require('ReactChildrenMutationWarningHook
 var ExecutionEnvironment = require('ExecutionEnvironment');
 
 var performanceNow = require('performanceNow');
+var invariant = require('invariant');
 var warning = require('warning');
 
+import type { ReactElement } from 'ReactElementType';
+import type { DebugID } from 'ReactInstanceType';
+
+type Hook = any;
```

Comment:
I created those two types in order to name them in the code even though they are not properly typed yet.